### ISSUE TITLE
Chokidar does not load and breaks silently the code into a Worker using ExpressJS and NodeJS worker thread

### ISSUE THREAD

**Issue by ladislaogarcia** [paulmillr/chokidar#1107]
I am trying to require chokidar in a  Worker Ising NodeJS worker thread into an ExpressJS application. 

If I try to do It, nothing happenes. There are neither errors or warning. 

if I remove the require line, the rest of the code works properly..

Does somebody knows what may be happenong?

**Comment by paulmillr** [comment]
Platform? Node version? Chok version?

**Comment by ladislaogarcia** [comment]
Sure, sorry. Here you have:
Platform: Windows10
Chokidar: v3.5.1 ( latest) 
NodeJS: 14
ExpressJS 4

If I try to do it into the server everything goes fine, but I am not able to make it works into a Worker asI said.

**Comment by paulmillr** [comment]
that's strange

**Comment by ladislaogarcia** [comment]
Sorry for I did not having responded until now.

Finally, I walked over another way and I had gotten what I was looking for. I'm not sure, because I have not had time to check it, but I think it was my fault. Your library was "accompanied" by a Worker and a socket connection. I think I got into trouble when my code started to creating the worker and socket over and over again ( I was using nodemon library too ). So the system had not in time to load your library before nodemon reload the expressjs server. so it could not work. It responded to the checkpoints into the code that was below it because it was less code and it had time to do it.

I have it in my personal to-do list :-) I will write here when knowing something. I close it until now. Then, I will reopen it, if you don´t mind it.

I take this opportunity to thank you for your library. Good idea !!!.
